src/sentry/templates/sentry/account/notifications.html

```diff
@@ -60,23 +60,22 @@
 
         {{ settings_form.subscribe_by_default|as_crispy_field }}
 
-        <hr />
-
         <h4>{% trans "Workflow" %}</h4>
 
-        <p>{% blocktrans %}Workflow notifications are separate from alerts and are generated for issue updates, such as:{% endblocktrans %}</p>
-
-        <ul>
-          <li>{% trans "Assignment" %}</li>
-          <li>{% trans "Comments" %}</li>
-          <li>{% trans "Regressions" %}</li>
-          <li>{% trans "Resolution" %}</li>
-        </ul>
-
+        <p>
+          {% blocktrans %}
+            Workflow notifications are separate from alerts and are generated
+            for issue updates, such as changes in issue assignment, changes to
+            resolution status (including regressions), and comments.
+          {% endblocktrans %}
+        </p>
         <p>
           {% blocktrans %}
             When workflow notifications are enabled for a project, you'll
             receive an email when your teammates perform any of these actions.
+            You'll be automatically be added as a participant on an issue by
```

Comment:
`You'll be automatically be added`

Reply:
😬

Reply:
0465cc55a5cb4d88ba0bbb22b9dc2e436582a80e